content/components/pin-input.md

```diff
@@ -0,0 +1,32 @@
+---
+title: PIN Input
+description: Allows users to input a sequence of one-character alphanumeric inputs.
+---
+
+<script>
+	import { APISection, ComponentPreview, PinInputDemo } from '@/components'
+	export let schemas;
+</script>
+
+<ComponentPreview name="pin-input-demo" comp="PinInput">
+
+<PinInputDemo slot="preview" />
+
+</ComponentPreview>
+
+## Structure
+
+```svelte
+<script lang="ts">
+	import { PinInput } from "bits-ui";
+</script>
+
+<PinInput.Root>
+	<PinInput.Input />
```

Comment:
should there be more than one Input here? this component is kinda special in the sense that you repeat the input multiple times to create just a single input. this way it's slightly misleading I think, you could be fooled into thinking that it's enough with one `<PinInput.Input />` and one `<PinInput.HiddenInput />`

Reply:
I think it's okay to repeat the `PinInput.Input` component since the pin is composed of multiple characters and each character is represented by an input field.

Reply:
updated to do just that, @huntabyte feel free to change if you disagree